src/app/shared/post-capture-card/post-capture-card.component.scss

```diff
@@ -7,21 +7,21 @@
 }
 
 mat-icon {
-  color: #000  ;
+  color: #000;
 }
 
-mat-card  {
-  padding-right: 0px;
-  padding-left: 0px;
-  width: 100%;    
+mat-card {
+  padding-right: 0;
+  padding-left: 0;
+  width: 100%;
 }
 
 mat-card .instagram {
```

Comment:
The class name `instagram` is not clear for its purpose. Rename it with a meaningful word, such as `fixed-ratio`.

Reply:
👍